lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -1,174 +1,174 @@
-"""
+'''
```

Comment:
Overzealous search and replace?  Docstrings should generally use 3 double quotes

Reply:
Yep, will be fixed